test/integration/concierge_impersonation_proxy_test.go

```diff
@@ -2051,6 +2051,7 @@ func createServiceAccountToken(ctx context.Context, t *testing.T, adminClient ku
 			Delete(context.Background(), serviceAccount.Name, metav1.DeleteOptions{}))
 	})
 
+	// TODO: What is this used for?
```

Comment:
What about this TODO?

Reply:
This usage of service account tokens is not relevant to this PR. Removed the TODO.